src/docfx/template/TemplateEngine.cs

```diff
@@ -211,7 +211,70 @@ private JObject LoadGlobalTokens()
 
             var jsonSchema = JsonUtility.DeserializeData<JsonSchema>(File.ReadAllText(schemaFilePath), new FilePath(schemaFilePath));
 
+            // temporary mapping, will retired after we support config it in UI portal
+            jsonSchema = LearnErrorMapping(mime, jsonSchema);
+
             return new JsonSchemaValidator(jsonSchema, forceError: true);
         }
+
+        private static JsonSchema LearnErrorMapping(string mime, JsonSchema jsonSchema)
+        {
+            string mappingPath = string.Empty;
+
+            switch (mime)
+            {
+                case "LearningPath":
+                    mappingPath = "data/schemas/learningpath-error-mapping.json";
+                    break;
+
+                case "Module":
+                    mappingPath = "data/schemas/module-error-mapping.json";
+                    break;
+
+                case "ModuleUnit":
+                    mappingPath = "data/schemas/moduleunit-error-mapping.json";
+                    break;
+            }
```

Comment:
Use c# 8.0 switch sugar to simplify?
            string mappingPath = mime switch
            {
                "LearningPath" => "data/schemas/learningpath-error-mapping.json",
                "Module" => "data/schemas/module-error-mapping.json",
                "ModuleUnit" => "data/schemas/moduleunit-error-mapping.json",
                _ => string.Empty,
            };

Reply:
Great idea.